app/service/collection.py

```diff
@@ -80,46 +92,53 @@ def validate_import_id(import_id: str) -> None:
 @db_session.with_transaction(__name__)
 @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
 def update(
-    collection: CollectionWriteDTO, db: Session = db_session.get_db()
+    import_id: str, collection: CollectionWriteDTO, db: Session = db_session.get_db()
 ) -> Optional[CollectionReadDTO]:
     """
     Updates a single collection with the values passed.
 
+    :param import_id str: The collection import_id to change.
     :param CollectionDTO collection: The DTO with all the values to change (or keep).
     :raises RepositoryError: raised on a database error.
     :raises ValidationError: raised should the import_id be invalid.
     :return Optional[CollectionDTO]: The updated collection or None if not updated.
     """
-    validate_import_id(collection.import_id)
 
     # TODO: implement changing of a collection's organisation
```

Comment:
Do we still need this TODO?

Reply:
I think so - as we may not want to do that at all